### ISSUE TITLE
No Result - what happened?

### ISSUE THREAD

**Issue by Steviey** [business-science/modeltime.gluonts#23]
Win 7/64bit , R 4.0.5, latest packages

I get no result. What can it be?

`library(modeltime.gluonts)
library(tidymodels)
library(tidyverse)

model_fit_deepar <- deep_ar(
  id                    = "id",
  freq                  = "M",
  prediction_length     = 24,
  lookback_length       = 36,
  epochs                = 10, 
  num_batches_per_epoch = 50,
  learn_rate            = 0.001,
  num_layers            = 2,
  dropout               = 0.10
) %>%
  set_engine("gluonts_deepar") %>%
  fit(value ~ ., training(m750_splits))

# Forecast with 95% Confidence Interval
modeltime_table( model_fit_deepar ) %>%
  modeltime_calibrate(new_data = testing(m750_splits)) %>%
  modeltime_forecast(
    new_data      = testing(m750_splits),
    actual_data   = m750,
    conf_interval = 0.95
  ) %>%
  plot_modeltime_forecast(.interactive = FALSE)`

--------------------------------
```
WARNING:gluonts.dataset.loader:Multiprocessing is not supported on Windows, num_workers will be set to None.
100%|##########| 50/50 [00:04<00:00, 10.73it/s, epoch=1/10, avg_epoch_loss=8.82]
100%|##########| 50/50 [00:03<00:00, 14.48it/s, epoch=2/10, avg_epoch_loss=7.65]
100%|##########| 50/50 [00:03<00:00, 14.33it/s, epoch=3/10, avg_epoch_loss=7.51]
100%|##########| 50/50 [00:03<00:00, 14.71it/s, epoch=4/10, avg_epoch_loss=7.33]
100%|##########| 50/50 [00:03<00:00, 14.32it/s, epoch=5/10, avg_epoch_loss=7.21]
100%|##########| 50/50 [00:03<00:00, 14.12it/s, epoch=6/10, avg_epoch_loss=7.07]
100%|##########| 50/50 [00:03<00:00, 14.73it/s, epoch=7/10, avg_epoch_loss=7.01]
100%|##########| 50/50 [00:03<00:00, 14.12it/s, epoch=8/10, avg_epoch_loss=6.94]
100%|##########| 50/50 [00:03<00:00, 14.53it/s, epoch=9/10, avg_epoch_loss=6.91]
100%|##########| 50/50 [00:03<00:00, 14.44it/s, epoch=10/10, avg_epoch_loss=6.88]learning rate from ``lr_scheduler`` has been overwritten by ``learning_rate`` in optimizer.
learning rate from ``lr_scheduler`` has been overwritten by ``learning_rate`` in optimizer.
```

**Comment by Steviey** [comment]
**UPDATE:**

Splitting the pipe up in pieces, gives me the desired result/plot with warnings...

```
library(modeltime.gluonts)
library(tidymodels)
library(tidyverse)

#View(m750_splits)

model_fit_deepar <- deep_ar(
  id                    = "id",
  freq                  = "M",
  prediction_length     = 24,
  lookback_length       = 36,
  epochs                = 10, 
  num_batches_per_epoch = 50,
  learn_rate            = 0.001,
  num_layers            = 2,
  dropout               = 0.10
) %>%
  set_engine("gluonts_deepar") %>%
  parsnip::fit(value ~ ., training(m750_splits))

  model_table <- modeltime_table(model_fit_deepar) 
  
  info<- calibration_table <- model_table %>%
    modeltime_calibrate(testing(m750_splits))
  
 #info<-calibration_table %>%
  #  modeltime_accuracy() %>%
   # table_modeltime_accuracy(.interactive = FALSE)
  
  fc<-info %>% modeltime_forecast(
    new_data      = testing(m750_splits),
    actual_data   = m750,
    conf_interval = 0.95
  )
  
  myPlot<- fc %>% plot_modeltime_forecast(.interactive = FALSE)
  
  print(fc)
  print(myPlot)

  old<-0
if(old>0){
# Forecast with 95% Confidence Interval
modeltime_table(model_fit_deepar) %>%
  modeltime_calibrate(new_data = testing(m750_splits)) %>%
  modeltime_forecast(
    new_data      = testing(m750_splits),
    actual_data   = m750,
    conf_interval = 0.95
  ) %>%
  plot_modeltime_forecast(.interactive = FALSE)
}


WARNING:gluonts.dataset.loader:Multiprocessing is not supported on Windows, num_workers will be set to None.
100%|##########| 50/50 [00:04<00:00, 10.47it/s, epoch=1/10, avg_epoch_loss=8.6]
100%|##########| 50/50 [00:03<00:00, 13.52it/s, epoch=2/10, avg_epoch_loss=7.74]
100%|##########| 50/50 [00:03<00:00, 13.93it/s, epoch=3/10, avg_epoch_loss=7.62]
100%|##########| 50/50 [00:03<00:00, 13.43it/s, epoch=4/10, avg_epoch_loss=7.48]
100%|##########| 50/50 [00:03<00:00, 13.12it/s, epoch=5/10, avg_epoch_loss=7.35]
100%|##########| 50/50 [00:03<00:00, 13.85it/s, epoch=6/10, avg_epoch_loss=7.21]
100%|##########| 50/50 [00:03<00:00, 14.28it/s, epoch=7/10, avg_epoch_loss=7.19]
100%|##########| 50/50 [00:03<00:00, 14.02it/s, epoch=8/10, avg_epoch_loss=7.06]
100%|##########| 50/50 [00:03<00:00, 14.08it/s, epoch=9/10, avg_epoch_loss=7.02]
100%|##########| 50/50 [00:03<00:00, 14.43it/s, epoch=10/10, avg_epoch_loss=6.9]# A tibble: 330 x 7
   .model_id .model_desc .key   .index     .value .conf_lo
       <int> <chr>       <fct>  <date>      <dbl>    <dbl>
 1        NA ACTUAL      actual 1990-01-01   6370       NA
 2        NA ACTUAL      actual 1990-02-01   6430       NA
 3        NA ACTUAL      actual 1990-03-01   6520       NA
 4        NA ACTUAL      actual 1990-04-01   6580       NA
 5        NA ACTUAL      actual 1990-05-01   6620       NA
 6        NA ACTUAL      actual 1990-06-01   6690       NA
 7        NA ACTUAL      actual 1990-07-01   6000       NA
 8        NA ACTUAL      actual 1990-08-01   5450       NA
 9        NA ACTUAL      actual 1990-09-01   6480       NA
10        NA ACTUAL      actual 1990-10-01   6820       NA
# ... with 320 more rows, and 1 more variable:
#   .conf_hi <dbl>
Warnmeldung:
In max(ids, na.rm = TRUE) :
  kein nicht-fehlendes Argument für max; gebe -Inf zurück
learning rate from ``lr_scheduler`` has been overwritten by ``learning_rate`` in optimizer.
learning rate from ``lr_scheduler`` has been overwritten by ``learning_rate`` in optimizer.
learning rate from ``lr_scheduler`` has been overwritten by ``learning_rate`` in optimizer.

```
